src/markdown-pages/install-with-kurl/cis-compliance.md

```diff
@@ -0,0 +1,68 @@
+---
+path: "/docs/install-with-kurl/cis-compliance"
+date: "2021-12-14"
+weight: 24
+linktitle: "CIS Compliance"
+title: "CIS Compliance"
+isAlpha: false
+---
+The kURL installer can be configured to be Center for Internet Security (CIS) compliant. This is an opt-in feature that is configured in the kURL specification by setting the `kurl.cisCompliance` field to `true`. For information about known limitations, see [Known Limitations](#known-limitations). For more information about CIS security compliance for Kubernetes, see the [CIS benchmark information](https://www.cisecurity.org/benchmark/kubernetes).
+
+The following settings are changed when `cisCompliance` is set to `true`:
+
+**Primary node configuration:**
```

Comment:
I feel like these configuration sections are missing a whole lot eg audit log and profiling settings. what are we looking to capture here?

Reply:
I can't find the comment, but I had asked in an earlier comment that we should be noting what is changed or turned on when this flag is used. In particular, I would imagine all of these settings behind the flag should be mentioned/logged so people know what the flag is doing.

Reply:
oh. i get it. these are only the ones behind the flag. lgtm